src/GraphQL.DataLoader.Tests/GraphQL.DataLoader.Tests.csproj

```diff
@@ -2,15 +2,15 @@
   <Import Project="../Tests.props" />
 
   <PropertyGroup Condition="'$(SingleTestPlatform)' == 'true'">
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SingleTestPlatform)' != 'true' and '$(OS)' == 'Windows_NT'">
-    <TargetFrameworks>net7.0;net6.0;net5.0;netcoreapp3.1;net48</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0;net6.0;net5.0;netcoreapp3.1;net48</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(SingleTestPlatform)' != 'true' and '$(OS)' != 'Windows_NT'">
-    <TargetFrameworks>net7.0;net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0;net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Did you consider moving these definitions to `Test.props` so there will be a single file to maintain?

Reply:
Well, it works if we only use `<TargetFrameworks>` and never `<TargetFramework>`

Reply:
Do you mean this doesn't work when defined in `Test.props`?
```xml
  <PropertyGroup Condition="'$(SingleTestPlatform)' == 'true'">
    <TargetFramework>net8.0</TargetFramework>
  </PropertyGroup>
```
And this in a test project
```
  <PropertyGroup>
    <SingleTestPlatform>true</SingleTestPlatform>
  </PropertyGroup>
```

Reply:
No, I mean to override the default set of frameworks with net8.0, such as for the api checks, you have to use `TargetFrameworks`.  Maybe you want to play with it some more; I didn't spend too much time on it.  I did try to add a condition so that if TargetFramework was set it would clear TargetFrameworks but that didn't work either.  the .NET build tools gave lots of errors.

Reply:
your example wouldn't work, however, because Tests.props is imported at the top of the props file for each test project.  we would have to move the import or add a test.targets or something

Reply:
> Tests.props is imported at the top of the props file

yeah, you are right